.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo"
```

Comment:
It would be beneficial to add the following package-ecosystem as well

```
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
    interval: "weekly"
```